examples/lambda/providers.tf

```diff
@@ -0,0 +1,33 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "4.25.0"
+    }
```

Comment:
I need to clean this up.. the `aws` provider is the only one that's actually required by this example.